components/ConvertSteps/ManageStep.js

```diff
@@ -0,0 +1,74 @@
+import React, { useEffect, useCallback, useState } from 'react'
+import PropTypes from 'prop-types'
+import Step from './Step'
+
+function ManageStep({
+  title,
+  number,
+  handleTx,
+  active,
+  onSuccess,
+  onError,
+  onHashCreation,
+  retry,
+}) {
+  const [stepStatus, setStepStatus] = useState('waiting')
+  const [hasBeenActivated, setHasBeenActivated] = useState(false)
+  const [retrying, setRetrying] = useState(false)
+  const [hash, setHash] = useState()
+
+  const handleStepSigning = useCallback(async () => {
+    setHasBeenActivated(true)
+
+    try {
+      setStepStatus('waiting')
```

Comment:
I'm wondering if it would be useful to define and export these as constants in the component themselves (both in `Step` and `Stepper`), so it feels less like we're using magic strings in parent components.